src/SchemaBuilder/ClassDecorator/RequireAllClassSchemaDecorator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ScrumWorks\OpenApiSchema\SchemaBuilder\ClassDecorator;
+
+use ReflectionClass;
+use ScrumWorks\OpenApiSchema\SchemaBuilder\ClassSchemaDecoratorInterface;
+use ScrumWorks\OpenApiSchema\ValueSchema\Builder\AbstractSchemaBuilder;
+use ScrumWorks\OpenApiSchema\ValueSchema\Builder\ObjectSchemaBuilder;
+
+class RequireAllClassSchemaDecorator implements ClassSchemaDecoratorInterface
```

Comment:
tohle neni dobry nazev... puvodni idea bylo rozdelit to na schema decorator ktery pracuje s anotacemi a rekneme defaultni ktery pracuje pouze s php informacemi...